Plugins/Flow.Launcher.Plugin.PluginsManager/PluginsManager.cs

```diff
@@ -78,7 +81,7 @@ internal List<Result> GetDefaultHotKeys()
                     IcoPath = icoPath,
                     Action = _ =>
                     {
-                        Context.API.ChangeQuery("pm install ");
+                        Context.API.ChangeQuery($"{Context.CurrentPluginMetadata.ActionKeyword} install ");
```

Comment:
this cant be using Settings.HotKeyInstall instead of just the word as well?

Reply:
Sure